src/lib/libraries/extensions/index.jsx

```diff
@@ -163,14 +126,14 @@ export default [
         smallDeviceImage: ev3MenuImage
     },
     {
-        name: 'LEGO Boost',
-        extensionId: 'boost',
-        iconURL: boostImage,
+        name: 'LEGO WeDo 2.0',
+        extensionId: 'wedo2',
+        iconURL: wedoImage,
         description: (
             <FormattedMessage
                 defaultMessage="Build with motors and sensors."
-                description="Description for the 'LEGO Boost' extension"
-                id="gui.extension.boost.description"
+                description="Description for the 'LEGO WeDo 2.0' extension"
```

Comment:
Did you want to add a note here similar to the note for the Google translate extension to not translate 'LEGO' ?